tests/regression/78-termination/25-leave-loop-goto-terminating.c

```diff
@@ -1,9 +1,9 @@
-// SKIP TERM PARAM: --set "ana.activated[+]" termination --set ana.activated[+] apron --enable ana.int.interval --set ana.apron.domain polyhedra
+// SKIP TODO TERM PARAM: --set "ana.activated[+]" termination --set ana.activated[+] apron --enable ana.int.interval --set ana.apron.domain polyhedra
```

Comment:
Why doesn't this work anymore?

Reply:
I'm not even sure why this worked in the first place, we even have the comment 

```
// Apron is not able to detect this
```

in the code. I think before it only worked because all proof obligations over loops where a signed variable is incremented in each iteration are trivial.